### PR TITLE
0.1.4

### PR DESCRIPTION
**Pull request by KeeganPBSA**
- Fixed issue where some accounts can't login due to mismatched keys.
- Now sending keys to the faucet in the order Owner | Active | Memo. This should always be the order that keys are sent to the faucet for registration.
- Fixed input box on account page
- Updated faucet URL
- Updated version of the wallet to 0.1.4
- Updated package files to fix dependency issues as well as build issues
- Updated build instructions in README.

### CODE REVIEW

File: dl/src/services/WalletService.js [hunk 1]
```diff
@@ -117,47 +117,63 @@ class WalletService {
...
-                last_modified: date,
-                chain_id: Apis.instance().chain_id
-            };
+        // The name of the wallet 
+        let walletName = iDB.getCurrentWalletName();
+
+        // Generate the keys using the account name and the password
+        let keys = KeyGeneratorService.generateKeys(account.name, password)
+
+        if (keys.owner.toPublicKey().toPublicKeyString() !== account.owner.key_auths[0][0]) {
```
Comment (mseaward) [inline dl/src/services/WalletService.js]: `.first()` vs `[0][0]`:
Any reason to use one or the other?

File: dl/src/services/WalletService.js [hunk 2]
```diff
@@ -117,47 +117,63 @@ class WalletService {
...
+            let tmp = keys.active
+            keys.active = keys.owner;
+            keys.owner = tmp;
+        }
```
Comment (mseaward) [inline dl/src/services/WalletService.js]: Perhaps:
```
let privateOwner, privateActive;
if (keys.owner.toPublicKey().toPublicKeyString() !== account.owner.key_auths.first()) {
    privateOwner = keys.Active;
    privateActive = keys.Owner;
}
```

File: dl/src/services/WalletService.js (hunk 1 above)
Comment (mseaward) [reply dl/src/services/WalletService.js]: if account is not an immutable then ignore my comment.

File: dl/src/services/WalletService.js (hunk 2 above)
Comment (mseaward) [reply dl/src/services/WalletService.js]: do not use `.first()` if account is not immutable

File: dl/src/services/WalletService.js (hunk 1 above)
Comment (KeeganPBSA) [reply dl/src/services/WalletService.js]: Account is not immutable. first() is not something we can use.

**Pull request merged**